core/src/main/resources/hudson/tasks/ArtifactArchiver/help-artifacts.html

```diff
@@ -5,4 +5,11 @@
    -- except that <tt>","</tt> (<em>comma</em>) is the only supported separator.
    The base directory is <a href='ws/'>the workspace</a>.
    You can only archive files that are located in your workspace.
+
+   An example on how to archive multiple artifacts from a specific folder:
+   <pre>archiveArtifacts: artifacts: 'target/*.jar'</pre>
```

Comment:
```suggestion
   <pre>archiveArtifacts artifacts: 'target/*.jar'</pre>
```
there is no : at the beginning